sysbuild/Kconfig.secureboot

```diff
@@ -0,0 +1,50 @@
+
+config SECURE_BOOT
+	bool "Use Secure Bootloader"
+	help
+	  Set this option to enable the first stage bootloader which
+	  verifies the signature of the app.
+
+if SECURE_BOOT
+
+choice SECURE_BOOT_CORE
+	bool "Secure boot SoC core"
+	default SECURE_BOOT_NETCORE
+	help
+	  SoC core where the first stage bootloader resides.
+
+config SECURE_BOOT_APPCORE
+	bool "Appcore"
+	help
+	  Locate first stage bootloader on application core.
+
+config SECURE_BOOT_NETCORE
+	bool "Netcore"
+	depends on SUPPORT_NETCORE
+	help
+	  Locate first stage bootloader on network core.
```

Comment:
This needs to support both (well both would actually be 3, app core b0, app core mcuboot as second stage bootloader, app core application, net core b0, net core application)